multi-node/aws/pkg/config/config.go

```diff
@@ -413,10 +504,18 @@ func getContextString(buf []byte, offset, lineCount int) string {
 	return string(buf[leftLimit:rightLimit])
 }
 
+type etcdInstance struct {
+	IPAddress   net.IP
+	SubnetIndex int
```

Comment:
Cluster upgrades can never support updating the Subnet or VPC parameters, unfortunately. I've had users ask about it- there are complex manual migration steps which cannot currently be automated :(

Now that you've asked, I'll make sure to make that explicit rather than just letting cloudformation roll back the update. https://github.com/coreos/coreos-kubernetes/pull/608#issuecomment-246045559